freight_bringup/config/analyzers.yaml

```diff
@@ -0,0 +1,67 @@
+pub_rate: 1.0
+base_path: '' # Optional, prepended to all diagnostic output
+analyzers:
```

Comment:
It appears we don't actually publish anything about battery to diagnostics at this time, so we can't aggregate it.